contracts/integrations/test/framework/assertions/decreaseStakingPoolOperatorShare.ts

```diff
@@ -17,21 +16,16 @@ export function validDecreaseStakingPoolOperatorShareAssertion(
 ): FunctionAssertion<[string, number], {}, void> {
     const { stakingWrapper } = deployment.staking;
 
-    return new FunctionAssertion<[string, number], {}, void>(
-        stakingWrapper.decreaseStakingPoolOperatorShare.bind(stakingWrapper),
-        {
-            after: async (_beforeInfo, _result: FunctionResult, args: [string, number], txData: Partial<TxData>) => {
-                const [poolId, expectedOperatorShare] = args;
+    return new FunctionAssertion<[string, number], {}, void>(stakingWrapper, 'decreaseStakingPoolOperatorShare', {
+        after: async (_beforeInfo, _result: FunctionResult, args: [string, number], _txData: Partial<TxData>) => {
+            const [poolId, expectedOperatorShare] = args;
 
-                logUtils.log(`decreaseStakingPoolOperatorShare(${poolId}, ${expectedOperatorShare})`);
+            // Checks that the on-chain pool's operator share has been updated.
+            const { operatorShare } = await stakingWrapper.getStakingPool(poolId).callAsync();
+            expect(operatorShare).to.bignumber.equal(expectedOperatorShare);
```

Comment:
These should eventually be replaced with logging, right?

Reply:
Do you just plan to do that in a separate PR?

Reply:
nah, if any of these `expect`s throw, the `Simulation` class will log the error in JSON format before re-throwing